app/MasterTemplate.hs

```diff
@@ -44,7 +44,7 @@ header page =
                     ! H.customAttribute "context" (textValue page)
         H.div ! A.class_ "nav-middle" $ do
             H.ul ! A.id "nav-links" $ do
-                H.li ! A.id "nav-graph" $ toLink "/graph" "Graph"
+                H.li ! A.id "nav-graph" $ toLink "/graph" "Graph▼"
```

Comment:
RIght, don't add the arrow here, since it'll look a bit awkward and show up all the time. Instead:

1. You can do what you were suggesting in your question and in the `GraphDropdown` component add a *class* to this element.
2. In the css, use the `::after` pseudoelement that @leenlab2 was mentioning to add the arrow after the link text. This will give you more control over the styling and positioning of the arrow.